boxsdk/object/upload_session.py

```diff
@@ -0,0 +1,167 @@
+# coding: utf-8
+from __future__ import unicode_literals, absolute_import
+
+import base64
+import hashlib
+import json
+
+from .base_object import BaseObject
+from ..config import API
+from ..pagination.chunked_upload_part_limit_offset_based_object_collection import ChunkedUploadPartLimitOffsetBasedObjectCollection
+
+
+class UploadSession(BaseObject):
```

Comment:
Naming should be consistent for the various objects. Either everything should start with `ChunkedUpload` or just `Upload` (my preference is the explicit `ChunkedUpload` but just upload could work).

But don't have an `UploadSession` that uses a class named `ChunkedUploadPartLimitOffsetBasedObjectCollection`. Also, the docstrings for creating upload sessions in `File` and `Folder` refer to this class name as `ChunkedUploadSession`, so make sure the names agree here and there.